src/main/java/org/launchcode/walkabout/controllers/ProfileController.java

```diff
@@ -1,21 +1,47 @@
 package org.launchcode.walkabout.controllers;
 
+import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpSession;
+import org.launchcode.walkabout.data.UserRepository;
+import org.launchcode.walkabout.models.User;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
+@RequestMapping("profile")
 public class ProfileController {
 
     //TODO: Add @Autowired annotation + repositories
 
-        @GetMapping("profile")
-        public String displayUserProfile(Model model, HttpSession session) {
-            model.addAttribute("loggedIn", session.getAttribute("user") != null);
+@Autowired
+   private UserRepository userRepository;
 
-            return "profile";
+@Autowired
+private AuthenticationController authenticationController;
+
+private User getCurrentUser(HttpServletRequest request) {
+    HttpSession session = request.getSession();
+    return authenticationController.getUserFromSession(session);
+}
+
+//        @GetMapping("profile")
```

Comment:
With this commented out, are you able to access the Profile Page once logged in? Also, which nav-bar do you see?   I had added this code in because under the fragments for the header, there is a th-block that is dependent on whether the user is "loggedIn" for the navigation bar to change from login/register to the profile/trips/log-a-trip

Reply:
I got this fixed. I was able to combine the Git Mapping for both and it is now displaying both the user name personalization I added and the "logged in" header.